laos-indexer/src/laos/tokenUriProcessor.ts

```diff
@@ -54,6 +54,15 @@ export async function processTokenURIs() {
     const tokenURIDataService = TokenURIDataService.getInstance(entityManager);
     await tokenURIDataService.updatePendingTokenUris();
   } catch (error) {
-    console.error('Error processing token URIs:', error);
+    console.error('Error processing token URIs:', formatError(error as Error));
   }
 }
+
+function formatError(error: Error): string {
```

Comment:
I'm scared that this method can fail sometimes under unexpected Errors. I suggest that we either add tests, or we put a try catch